test/manual/amp-sidebar-1.0.amp.html

```diff
@@ -146,7 +146,7 @@
 
     <!-- Toolbar One, basic usage, toolbar-only -->
     <!-- Could center with CSS justify-content: space-between, but trying to be minimal -->
-    <nav toolbar="(min-width: 0px)" class="ampstart-hamburger-nav ampstart-navbar-bg" target="hamburger-toolbar" toolbar-only>
+    <nav toolbar="(min-width: 0px)" target="hamburger-toolbar" class="ampstart-hamburger-nav ampstart-navbar-bg" toolbar-only>
```

Comment:
Yes, it is for both, and I created an issue here: https://github.com/ampproject/amphtml/issues/10432